spinegeneric/cli/manual_correction.py

```diff
@@ -54,7 +56,9 @@ def get_parser():
             - sub-amu01_T2star_rms.nii.gz
             FILES_LABEL:
             - sub-amu01_T1w_RPI_r.nii.gz
-            - sub-amu02_T1w_RPI_r.nii.gz\n
+            - sub-amu02_T1w_RPI_r.nii.gz
+            NAME:
+            - John Doe\n
```

Comment:
These two lines should not be included in input yml file, eventually. It causes error when I include them - `manual_correction.py` expects only filenames in input yml file.

Reply:
oops! forgot about this, fixed in 5be1ae2fe193286cb856b44a5280536495887208
thanks!